user_management/doc/User_management.md

```diff
@@ -281,6 +281,52 @@ will return public user information
 
 </details>
 
+## `/user/id-list/`
+
+### Get a list of user ids
+
+will return a list of user ids
+
+<details>
+ <summary><code>POST</code><code><b>/user/id_list/</b></code></summary>
+
+### Parameters
+
+#### Body
+
+> ``` javascript
+> 
+> {
+>     "id_list": ["1", "2", "3"]
```

Comment:
if you take a list of integers, you need to remove the double quotes around each element in the list